docs/guide.md

```diff
@@ -0,0 +1,210 @@
+# Getting Started with PatternPack
```

Comment:
So I went through and looked at two frameworks I know have good documentation:

* [Getting Started with Rails](http://guides.rubyonrails.org/getting_started.html)
* [Getting Started with React](https://facebook.github.io/react/docs/getting-started.html)
* [Installing Laravel](http://laravel.com/docs/5.1/installation)

All of these use "you". I'm not sure how you'd write this tutorial without saying that.